File: mvps/nodejs/src/results/results.controller.ts
import { Controller, Get } from '@nestjs/common';
import { ResultsService } from './results.service';

@Controller('v1/results')
export class ResultsController {
  constructor(private readonly resultsService: ResultsService) {}

  @Get()
  getResults() {
    return this.resultsService.getResults();
  }
}

File: mvps/java/src/main/java/com/meetup/beervote/web/ResultsController.java
package com.meetup.beervote.web;

import com.meetup.beervote.dto.ApiDtos.ResultsResponse;
import com.meetup.beervote.results.ResultsService;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/v1/results")
public class ResultsController {

    private final ResultsService resultsService;

    public ResultsController(ResultsService resultsService) {
        this.resultsService = resultsService;
    }

    @GetMapping
    public ResultsResponse results() {
        return resultsService.getResults();
    }
}
